modules/customize-certificates-add-service-serving-configmap.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-configmap-{context}"]
+= Add a service certificate to a ConfigMap
+
+To secure communication to services that consume a ConfigMap, 
+have the cluster generate a signed serving certificate and key pair into the
+ConfigMap. To do this, set the 
+`service.beta.openshift.io/inject-cabundle=true` 
+annotation on your ConfigMap. A certificate will be generated, which allows
+consumers to trust the `service-ca.crt` in their TLS client configuration,
```

Comment:
My understanding is that this would be anything that uses the ConfigMap. This could be a Pod, a controller, or anything else that references the ConfigMap. I'm leaving this one as `consumers` for now.

I've added `generated` before `service-ca.crt` to indicate this certificate is created and does not need to be supplied. Hopefully the procedure below also indicates this, indicating that the `service-ca.crt` has been added to the ConfigMap by the previous step.